include/ebpf_store_helper.h

```diff
@@ -3,350 +3,79 @@
 
 #pragma once
 
+#ifndef USER_MODE
+#include "framework.h"
+#else
+#include "ebpf_utilities.h"
+#endif
+
 #include "ebpf_program_types.h"
-#include "ebpf_registry_helper.h"
 #include "ebpf_windows.h"
 
-#ifdef USER_MODE
-extern ebpf_registry_key_t ebpf_root_registry_key;
-#endif
+#define GUID_STRING_LENGTH 38 // not including the null terminator.
 
-static uint32_t
-_ebpf_store_open_or_create_provider_registry_key(_Out_ ebpf_registry_key_t* provider_key)
+#ifdef __cplusplus
+extern "C"
 {
-    __return_type status = _SUCCESS;
-    ebpf_registry_key_t root_key = NULL;
-    *provider_key = NULL;
+#endif
 
-    // Open (or create) root eBPF registry path.
 #ifndef USER_MODE
-    status = create_registry_key(NULL, EBPF_ROOT_REGISTRY_PATH, REG_CREATE_FLAGS, &root_key);
+    typedef HANDLE ebpf_store_key_t;
 #else
-    status = create_registry_key(ebpf_root_registry_key, EBPF_ROOT_RELATIVE_PATH, REG_CREATE_FLAGS, &root_key);
+typedef HKEY ebpf_store_key_t;
```

Comment:
Make indentation consistent between lines 23 and 25

Reply:
I've already attempted to do that, but the format checking would bring it back. I can make this checking to ignore formatting as an exception.